public/electron.js

```diff
@@ -1,56 +1,34 @@
 const {app, BrowserWindow} = require('electron');
-const path = require('path');
-const url = require('url');
-
-// Keep a global reference of the window object, if you don't, the window will
-// be closed automatically when the JavaScript object is garbage collected.
-let win;
-
+const {join} = require('path');
+const {format} = require('url');
+let isDev = process.env.NODE_ENV === 'development';
+let mainWindow;
```

Comment:
Could you please return all the comments that were in this file? They might be useful for further support.